frontend/app/(tabs)/create.tsx

```diff
@@ -92,6 +96,7 @@ export default function Create() {
 
   return (
     <SafeAreaView>
+      <EditPostDialog />
```

Comment:
This should be moved into the `Post` component since we only want to update if the current user is the author and if its already posted.